### PR TITLE
Topics: Merged topics #9

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary

Merges 30 topics.

> [!NOTE]
> 1 topic was removed in this PR and two `ref` shortcodes were updated

- [x] **digitalgov-search** removed
- [x] content/news/2014/09/2014-09-12-digitalgov-gov-six-months-later.md
- [x] content/news/2016/11/2016-11-28-gsa-presents-grace-hopper-day-hackathon.md

<details>
<summary>Topics merged into new topics</summary>

| old topic                                         | new topic            |
| ------------------------------------------------- | -------------------- |
| aoi                                               | user-experience      |
| code                                              | software-engineering |
| community-ux                                      | user-experience      |
| cybersecurity-national-action-plan                | security             |
| developer-hub                                     | software-engineering |
| devops                                            | software-engineering |
| digitalgov-search                                 | search               |
| digitalgov-user-experience-program                | user-experience      |
| federal-risk-and-authorization-management-program | security             |
| federal-user-experience-community                 | user-experience      |
| fedramp                                           | security             |
| first-click-testing                               | usability            |
| github                                            | software-engineering |
| hackathon                                         | software-engineering |
| https                                             | security             |
| ios                                               | software-engineering |
| json                                              | software-engineering |
| national-cyber-security-awareness-month           | security             |
| national-day-of-civic-hacking                     | software-engineering |
| saas                                              | software-engineering |
| social-media-metrics                              | social-media         |
| socialgov                                         | social-media         |
| socialgov-community                               | social-media         |
| tos                                               | terms-of-service     |
| ugc                                               | social-media         |
| us-digital-registry                               | social-media         |
| usability-case-study                              | usability            |
| usability-starter-kit                             | usability            |
| usability-testing                                 | usability            |
| user-experience-community-of-practice             | user-experience      |

</details>

### Solution

- [x] replace `old topic` with `new topic` in front matter
- [x] delete `old topic` folder in `content/topics`
- [x] update `ref` shortcodes to use new topic

---

### Dev Checklist

- [ ] PR has correct labels
- [ ] A11y testing (voice over testing, meets WCAG, run axe tools)
- [ ] Code is formatted properly

### CODE REVIEW

**Comment by clmedders** [comment]
lgtm 
i testing that old topics are not showing up with `^\s*-\s* (aoi | code | community-ux | cybersecurity-national-action-plan | developer-hub | devops | digitalgov-search | digitalgov-user-experience-program | federal-risk-and-authorization-management-program |  federal-user-experience-community | fedramp | frist-click-testing | github | hackathon | https | ios | json | national-cyber-security-awareness-month | national-day-of-civic-hacking | saas | social-media-metrics | socialgov | socialgov-community | tos | ugc | us-digital-registry | usability-case-study | usability-starter-kit | usability-testing | user-experience-community-of-practice)`

also that topics index was not there 

also cleaned up front matter from what i saw of repeat topics and empty lines

**Review by clmedders — APPROVED** [review]

**Review by mejiaj — APPROVED** [review]
LGTM. No references found to old topics in frontmatter or code.

**Regex tested**

`^\s*/topics/s* (aoi|code|community-ux|cybersecurity-national-action-plan|developer-hub|devops|digitalgov-search|digitalgov-user-experience-program|federal-risk-and-authorization-management-program|federal-user-experience-community|fedramp|first-click-testing|github|hackathon|https|ios|json|national-cyber-security-awareness-month|national-day-of-civic-hacking|saas|social-media-metrics|socialgov|socialgov-community|tos|ugc|us-digital-registry|usability-case-study|usability-starter-kit|usability-testing|user-experience-community-of-practice)$`

**Review by clmedders — APPROVED** [review]

**Review by mejiaj — DISMISSED** [review]
Looks good. No conflicts or references to old topics in frontmatter or content.

File: themes/digital.gov/layouts/partials/core/get_author_image.html
Comment (ToniBonittoGSA) [inline themes/digital.gov/layouts/partials/core/get_author_image.html:1]: Change `software-engineering` back to `github`

File: content/news/2016/09/2016-09-28-the-new-vote-gov-leaner-faster-and-multi-lingual.md
Comment (ToniBonittoGSA) [inline content/news/2016/09/2016-09-28-the-new-vote-gov-leaner-faster-and-multi-lingual.md:1]: restore `content-strategy`

File: content/events/2021/06/2021-06-03-2021-user-experience-summit.md
Comment (ToniBonittoGSA) [inline content/events/2021/06/2021-06-03-2021-user-experience-summit.md:1]: Remove duplicates

File: content/news/2013/03/2013-03-22-regulations-gov-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2013/03/2013-03-22-regulations-gov-usability-case-study.md:1]: Remove duplicates

File: content/news/2013/04/2013-04-29-atf-gov-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2013/04/2013-04-29-atf-gov-usability-case-study.md:1]: Remove duplicates

File: content/news/2013/07/2013-07-02-business-usa-gov-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2013/07/2013-07-02-business-usa-gov-usability-case-study.md:1]: Remove duplicates

File: content/news/2014/04/2014-04-04-heatmapping-tools-show-whats-hot-on-your-pages.md
Comment (ToniBonittoGSA) [inline content/news/2014/04/2014-04-04-heatmapping-tools-show-whats-hot-on-your-pages.md:1]: Remove duplicate

File: content/news/2014/06/2014-06-06-consumer-action-handbook-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2014/06/2014-06-06-consumer-action-handbook-usability-case-study.md:1]: Remove duplicates

File: content/news/2014/10/2014-10-06-user-acceptance-testing-versus-usability-testing-whats-the-dif.md
Comment (ToniBonittoGSA) [inline content/news/2014/10/2014-10-06-user-acceptance-testing-versus-usability-testing-whats-the-dif.md:1]: Remove duplicate

File: content/news/2014/11/2014-11-13-world-usability-day-engagement.md
Comment (ToniBonittoGSA) [inline content/news/2014/11/2014-11-13-world-usability-day-engagement.md:1]: remove duplicates

File: content/news/2015/02/2015-02-06-top-task-usability-design-for-your-users.md
Comment (ToniBonittoGSA) [inline content/news/2015/02/2015-02-06-top-task-usability-design-for-your-users.md:1]: remove duplicates

File: content/news/2015/02/2015-02-25-avoid-weak-links-in-your-digital-chain.md
Comment (ToniBonittoGSA) [inline content/news/2015/02/2015-02-25-avoid-weak-links-in-your-digital-chain.md:1]: remove duplicates

File: content/news/2015/11/2015-11-09-how-ux-effects-change-in-government-one-test-one-customer-survey-at-a-time.md
Comment (ToniBonittoGSA) [inline content/news/2015/11/2015-11-09-how-ux-effects-change-in-government-one-test-one-customer-survey-at-a-time.md:1]: Remove duplicates

File: content/news/2015/11/2015-11-20-ux-vs-cx-whats-the-dif-part-2.md
Comment (ToniBonittoGSA) [inline content/news/2015/11/2015-11-20-ux-vs-cx-whats-the-dif-part-2.md:1]: Remove duplicates

File: content/news/2016/02/2016-02-10-paying-incentives-for-federal-user-research.md
Comment (ToniBonittoGSA) [inline content/news/2016/02/2016-02-10-paying-incentives-for-federal-user-research.md:1]: Remove duplicate

File: content/news/2021/02/2021-02-08-call-for-participation-2021-government-ux-summit.md
Comment (ToniBonittoGSA) [inline content/news/2021/02/2021-02-08-call-for-participation-2021-government-ux-summit.md:1]: remove duplicate

File: content/news/2014/12/2014-12-16-trends-on-tuesday-smartphone-users-still-use-the-phone-for-talking.md
Comment (ToniBonittoGSA) [inline content/news/2014/12/2014-12-16-trends-on-tuesday-smartphone-users-still-use-the-phone-for-talking.md:1]: Remove the one without dashes

File: content/news/2015/07/2015-07-17-is-it-time-to-try-medium-new-federal-friendly-terms-of-service.md
Comment (ToniBonittoGSA) [inline content/news/2015/07/2015-07-17-is-it-time-to-try-medium-new-federal-friendly-terms-of-service.md:1]: Remove duplicate

File: content/news/2015/08/2015-08-20-facts-on-the-federal-compatible-terms-of-service-agreement-for-yelp.md
Comment (ToniBonittoGSA) [inline content/news/2015/08/2015-08-20-facts-on-the-federal-compatible-terms-of-service-agreement-for-yelp.md:1]: Remove duplicates

File: content/events/2021/04/2021-04-20-social-media-spring-session-2021.md
Comment (ToniBonittoGSA) [inline content/events/2021/04/2021-04-20-social-media-spring-session-2021.md:1]: remove duplicates

File: content/events/2021/04/2021-04-27-deep-fakes-and-social-media.md
Comment (ToniBonittoGSA) [inline content/events/2021/04/2021-04-27-deep-fakes-and-social-media.md:1]: Remove duplicate

File: content/news/2012/11/2012-11-30-socialgov-summit-highlights-accessibility-challenges.md
Comment (ToniBonittoGSA) [inline content/news/2012/11/2012-11-30-socialgov-summit-highlights-accessibility-challenges.md:1]: Remove duplicate

File: content/news/2013/04/2013-04-19-social-media-metrics-for-federal-agencies.md
Comment (ToniBonittoGSA) [inline content/news/2013/04/2013-04-19-social-media-metrics-for-federal-agencies.md:1]: Remove duplicate

File: content/news/2013/04/2013-04-25-government-must-respond-rapidly-to-social-media-hacking.md
Comment (ToniBonittoGSA) [inline content/news/2013/04/2013-04-25-government-must-respond-rapidly-to-social-media-hacking.md:1]: Remove duplicate

File: content/news/2013/05/2013-05-31-twitters-two-step-verification-process.md
Comment (ToniBonittoGSA) [inline content/news/2013/05/2013-05-31-twitters-two-step-verification-process.md:1]: remove duplicate

File: content/news/2013/06/2013-06-26-5-myths-about-social-media-accessibility-2.md
Comment (ToniBonittoGSA) [inline content/news/2013/06/2013-06-26-5-myths-about-social-media-accessibility-2.md:1]: Remove duplicates

File: content/news/2013/09/2013-09-18-best-week-in-socialgov-ever-week-1.md
Comment (ToniBonittoGSA) [inline content/news/2013/09/2013-09-18-best-week-in-socialgov-ever-week-1.md:1]: Remove duplicate

File: content/news/2014/01/2014-01-10-advancing-forward-best-of-social-media-in-2013-and-beyond.md
Comment (ToniBonittoGSA) [inline content/news/2014/01/2014-01-10-advancing-forward-best-of-social-media-in-2013-and-beyond.md:1]: remove duplicate

File: content/news/2014/01/2014-01-28-innovation-means-more-better-cost-effective-digital-training.md
Comment (ToniBonittoGSA) [inline content/news/2014/01/2014-01-28-innovation-means-more-better-cost-effective-digital-training.md:1]: remove duplicate

File: content/news/2014/02/2014-02-10-managing-measuring-and-archiving-with-hootsuite.md
Comment (ToniBonittoGSA) [inline content/news/2014/02/2014-02-10-managing-measuring-and-archiving-with-hootsuite.md:1]: remove duplicate

File: content/news/2015/02/2015-02-12-ifttt-combines-social-media-mobile-and-internet-of-things-for-government.md
Comment (ToniBonittoGSA) [inline content/news/2015/02/2015-02-12-ifttt-combines-social-media-mobile-and-internet-of-things-for-government.md:1]: remove duplicate

File: content/news/2015/06/2015-06-05-monthly-theme-building-evaluating-and-improving-government-services-through-social-media.md
Comment (ToniBonittoGSA) [inline content/news/2015/06/2015-06-05-monthly-theme-building-evaluating-and-improving-government-services-through-social-media.md:1]: remove duplicate

File: content/news/2015/08/2015-08-25-digitalgov-podcast-the-secrets-behind-cia-social-media.md
Comment (ToniBonittoGSA) [inline content/news/2015/08/2015-08-25-digitalgov-podcast-the-secrets-behind-cia-social-media.md:1]: remove duplicate

File: content/news/2016/07/2016-07-11-how-to-leverage-user-generated-content-to-resonate-with-your-audience.md
Comment (ToniBonittoGSA) [inline content/news/2016/07/2016-07-11-how-to-leverage-user-generated-content-to-resonate-with-your-audience.md:1]: remove duplicate

File: content/events/2018/06/2018-06-07-free-open-source-software-in-government-with-codemil.md
Comment (ToniBonittoGSA) [inline content/events/2018/06/2018-06-07-free-open-source-software-in-government-with-codemil.md:1]: remove duplicate

File: content/events/2020/10/2020-10-08-federal-source-code-summit-building-coding.md
Comment (ToniBonittoGSA) [inline content/events/2020/10/2020-10-08-federal-source-code-summit-building-coding.md:1]: remove duplicate

File: content/news/2016/11/2016-11-21-mobile-development-and-testing-with-chrome-devtools.md
Comment (ToniBonittoGSA) [inline content/news/2016/11/2016-11-21-mobile-development-and-testing-with-chrome-devtools.md:1]: Remove duplicate

File: content/news/2017/06/2017-06-27-the-journey-to-sharing-americas-code.md
Comment (ToniBonittoGSA) [inline content/news/2017/06/2017-06-27-the-journey-to-sharing-americas-code.md:1]: remove duplicate

File: content/events/2018/06/2018-06-14-usability-testing-with-steve-krug.md
Comment (ToniBonittoGSA) [inline content/events/2018/06/2018-06-14-usability-testing-with-steve-krug.md:1]: remove duplicate

File: content/news/2013/02/2013-02-19-contractor-vehicle-navigator-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2013/02/2013-02-19-contractor-vehicle-navigator-usability-case-study.md:1]: remove duplicates

File: content/news/2013/06/2013-06-19-army-mil-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2013/06/2013-06-19-army-mil-usability-case-study.md:1]: remove duplicates

File: content/news/2013/07/2013-07-02-fueleconomy-gov-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2013/07/2013-07-02-fueleconomy-gov-usability-case-study.md:1]: Remove duplicates

File: content/news/2014/04/2014-04-14-noaa-national-ocean-service-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2014/04/2014-04-14-noaa-national-ocean-service-usability-case-study.md:1]: remove duplicates

File: content/news/2015/04/2015-04-10-usability-testing-with-people-who-have-vision-impairment-is-difficult-reality-or-perception.md
Comment (ToniBonittoGSA) [inline content/news/2015/04/2015-04-10-usability-testing-with-people-who-have-vision-impairment-is-difficult-reality-or-perception.md:1]: Remove duplicates

File: content/news/2015/04/2015-04-21-trends-on-tuesday-the-total-market-approach.md
Comment (ToniBonittoGSA) [inline content/news/2015/04/2015-04-21-trends-on-tuesday-the-total-market-approach.md:1]: remove duplicate

File: content/news/2021/08/2021-08-13-a-user-centered-approach-simplicity-and-complexity-in-a-homepage-graphic.md
Comment (ToniBonittoGSA) [inline content/news/2021/08/2021-08-13-a-user-centered-approach-simplicity-and-complexity-in-a-homepage-graphic.md:1]: remove duplicate

File: content/news/2022/06/2022-06-13-dhs-usability-testing-kit.md
Comment (ToniBonittoGSA) [inline content/news/2022/06/2022-06-13-dhs-usability-testing-kit.md:1]: remove duplicate

File: content/news/2023/10/2023-10-11-5-things-we-learned-from-our-scams-wizard-usability-test.md
Comment (ToniBonittoGSA) [inline content/news/2023/10/2023-10-11-5-things-we-learned-from-our-scams-wizard-usability-test.md:1]: remove duplicate

File: content/resources/department-of-homeland-security-usability-testing-kit.md
Comment (ToniBonittoGSA) [inline content/resources/department-of-homeland-security-usability-testing-kit.md:1]: remove duplicate

File: content/resources/digitalgov-user-experience-program-results-and-case-studies.md
Comment (ToniBonittoGSA) [inline content/resources/digitalgov-user-experience-program-results-and-case-studies.md:1]: remove duplicate

File: content/resources/government-usability-case-studies.md
Comment (ToniBonittoGSA) [inline content/resources/government-usability-case-studies.md:1]: remove duplicate

File: content/events/2019/05/2019-05-15-2019-government-ux-summit.md
Comment (ToniBonittoGSA) [inline content/events/2019/05/2019-05-15-2019-government-ux-summit.md:1]: remove duplicate

File: content/news/2014/06/2014-06-09-facebook-increases-public-service-verification-to-improve-citizen-engagement-2.md
Comment (ToniBonittoGSA) [inline content/news/2014/06/2014-06-09-facebook-increases-public-service-verification-to-improve-citizen-engagement-2.md:1]: remove duplicate

File: content/news/2014/11/2014-11-25-help-shape-public-participation.md
Comment (ToniBonittoGSA) [inline content/news/2014/11/2014-11-25-help-shape-public-participation.md:1]: remove duplicate

File: content/news/2016/04/2016-04-05-additional-data-options-with-dap.md
Comment (ToniBonittoGSA) [inline content/news/2016/04/2016-04-05-additional-data-options-with-dap.md:1]: remove duplicate

File: content/news/2013/03/2013-03-21-saferbus-mobile-usability-case-study.md
Comment (ToniBonittoGSA) [inline content/news/2013/03/2013-03-21-saferbus-mobile-usability-case-study.md:1]: remove duplicates

File: content/news/2017/09/2017-09-28-finding-usability-testers-tips-from-an-army-recruiter.md
Comment (ToniBonittoGSA) [inline content/news/2017/09/2017-09-28-finding-usability-testers-tips-from-an-army-recruiter.md:1]: remove duplicate

File: content/events/2022/03/2022-03-04-plain-language-compassion-help-people-in-crisis.md
Comment (ToniBonittoGSA) [inline content/events/2022/03/2022-03-04-plain-language-compassion-help-people-in-crisis.md:1]: Restore research

File: content/events/2022/09/2022-09-08-dhs-usability-testing-kit-so-easy-anyone-can-do-it.md
Comment (ToniBonittoGSA) [inline content/events/2022/09/2022-09-08-dhs-usability-testing-kit-so-easy-anyone-can-do-it.md:1]: Remove `user-testing-and-research` (is being merged into `research` which is already there).

File: content/news/2013/05/2013-05-25-functionality-and-usability-testing-resources.md
Comment (ToniBonittoGSA) [inline content/news/2013/05/2013-05-25-functionality-and-usability-testing-resources.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2013/10/2013-10-29-which-devices-should-i-test-with.md
Comment (ToniBonittoGSA) [inline content/news/2013/10/2013-10-29-which-devices-should-i-test-with.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2013/11/2013-11-21-mobile-product-compatibility-and-functional-testing-101.md
Comment (ToniBonittoGSA) [inline content/news/2013/11/2013-11-21-mobile-product-compatibility-and-functional-testing-101.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2014/05/2014-05-20-packaging-up-api-usability-testing-for-agency-reuse.md
Comment (ToniBonittoGSA) [inline content/news/2014/05/2014-05-20-packaging-up-api-usability-testing-for-agency-reuse.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2014/05/2014-05-20-trends-on-tuesday-larger-screen-smartphones-see-accelerated-shipments.md
Comment (ToniBonittoGSA) [inline content/news/2014/05/2014-05-20-trends-on-tuesday-larger-screen-smartphones-see-accelerated-shipments.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2014/05/2014-05-21-the-road-to-better-websites-gets-easier-with-usability-walkthroughs.md
Comment (ToniBonittoGSA) [inline content/news/2014/05/2014-05-21-the-road-to-better-websites-gets-easier-with-usability-walkthroughs.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2014/05/2014-05-22-how-to-make-a-mobile-paper-prototype.md
Comment (ToniBonittoGSA) [inline content/news/2014/05/2014-05-22-how-to-make-a-mobile-paper-prototype.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2015/06/2015-06-09-the-emulator-dilemma-can-mobile-device-testing-be-completed-without-mobile-devices.md
Comment (ToniBonittoGSA) [inline content/news/2015/06/2015-06-09-the-emulator-dilemma-can-mobile-device-testing-be-completed-without-mobile-devices.md:1]: Change `user-testing-and-research` to `research`.

File: content/news/2016/05/2016-05-02-redesigning-we-the-people.md
Comment (ToniBonittoGSA) [inline content/news/2016/05/2016-05-02-redesigning-we-the-people.md:1]: Change `user-testing-and-research` to `research`.

File: content/events/2019/05/2019-05-15-2019-government-ux-summit.md
Comment (clmedders) [reply content/events/2019/05/2019-05-15-2019-government-ux-summit.md:1]: Removed

File: content/events/2021/04/2021-04-20-social-media-spring-session-2021.md
Comment (clmedders) [reply content/events/2021/04/2021-04-20-social-media-spring-session-2021.md:1]: Removed

File: content/events/2021/06/2021-06-03-2021-user-experience-summit.md
Comment (clmedders) [reply content/events/2021/06/2021-06-03-2021-user-experience-summit.md:1]: Removed

File: content/events/2022/09/2022-09-08-dhs-usability-testing-kit-so-easy-anyone-can-do-it.md
Comment (clmedders) [reply content/events/2022/09/2022-09-08-dhs-usability-testing-kit-so-easy-anyone-can-do-it.md:1]: Removed

File: content/news/2012/11/2012-11-30-socialgov-summit-highlights-accessibility-challenges.md
Comment (clmedders) [reply content/news/2012/11/2012-11-30-socialgov-summit-highlights-accessibility-challenges.md:1]: Removed

File: content/news/2013/02/2013-02-19-contractor-vehicle-navigator-usability-case-study.md
Comment (clmedders) [reply content/news/2013/02/2013-02-19-contractor-vehicle-navigator-usability-case-study.md:1]: Updated

File: content/news/2013/03/2013-03-21-saferbus-mobile-usability-case-study.md
Comment (clmedders) [reply content/news/2013/03/2013-03-21-saferbus-mobile-usability-case-study.md:1]: updated

File: content/news/2013/03/2013-03-22-regulations-gov-usability-case-study.md
Comment (clmedders) [reply content/news/2013/03/2013-03-22-regulations-gov-usability-case-study.md:1]: updated

File: content/news/2013/04/2013-04-19-social-media-metrics-for-federal-agencies.md
Comment (clmedders) [reply content/news/2013/04/2013-04-19-social-media-metrics-for-federal-agencies.md:1]: Removed

File: content/news/2013/04/2013-04-25-government-must-respond-rapidly-to-social-media-hacking.md
Comment (clmedders) [reply content/news/2013/04/2013-04-25-government-must-respond-rapidly-to-social-media-hacking.md:1]: Removed

File: content/news/2013/04/2013-04-29-atf-gov-usability-case-study.md
Comment (clmedders) [reply content/news/2013/04/2013-04-29-atf-gov-usability-case-study.md:1]: updated

File: content/news/2013/05/2013-05-25-functionality-and-usability-testing-resources.md
Comment (clmedders) [reply content/news/2013/05/2013-05-25-functionality-and-usability-testing-resources.md:1]: updated

File: content/news/2013/05/2013-05-31-twitters-two-step-verification-process.md
Comment (clmedders) [reply content/news/2013/05/2013-05-31-twitters-two-step-verification-process.md:1]: Removed

File: content/news/2013/06/2013-06-19-army-mil-usability-case-study.md
Comment (clmedders) [reply content/news/2013/06/2013-06-19-army-mil-usability-case-study.md:1]: updated

File: content/news/2013/06/2013-06-26-5-myths-about-social-media-accessibility-2.md
Comment (clmedders) [reply content/news/2013/06/2013-06-26-5-myths-about-social-media-accessibility-2.md:1]: Removed

Comment (clmedders) [reply content/news/2013/06/2013-06-26-5-myths-about-social-media-accessibility-2.md:1]: Removed

File: content/news/2013/07/2013-07-02-business-usa-gov-usability-case-study.md
Comment (clmedders) [reply content/news/2013/07/2013-07-02-business-usa-gov-usability-case-study.md:1]: updated

File: content/news/2013/07/2013-07-02-fueleconomy-gov-usability-case-study.md
Comment (clmedders) [reply content/news/2013/07/2013-07-02-fueleconomy-gov-usability-case-study.md:1]: updated

File: content/news/2013/09/2013-09-18-best-week-in-socialgov-ever-week-1.md
Comment (clmedders) [reply content/news/2013/09/2013-09-18-best-week-in-socialgov-ever-week-1.md:1]: Removed

File: content/news/2013/10/2013-10-29-which-devices-should-i-test-with.md
Comment (clmedders) [reply content/news/2013/10/2013-10-29-which-devices-should-i-test-with.md:1]: updated

File: content/news/2013/11/2013-11-21-mobile-product-compatibility-and-functional-testing-101.md
Comment (clmedders) [reply content/news/2013/11/2013-11-21-mobile-product-compatibility-and-functional-testing-101.md:1]: updated

File: content/news/2014/01/2014-01-10-advancing-forward-best-of-social-media-in-2013-and-beyond.md
Comment (clmedders) [reply content/news/2014/01/2014-01-10-advancing-forward-best-of-social-media-in-2013-and-beyond.md:1]: updated

File: content/news/2014/01/2014-01-28-innovation-means-more-better-cost-effective-digital-training.md
Comment (clmedders) [reply content/news/2014/01/2014-01-28-innovation-means-more-better-cost-effective-digital-training.md:1]: updated

File: content/news/2014/02/2014-02-10-managing-measuring-and-archiving-with-hootsuite.md
Comment (clmedders) [reply content/news/2014/02/2014-02-10-managing-measuring-and-archiving-with-hootsuite.md:1]: updated

File: content/news/2014/04/2014-04-04-heatmapping-tools-show-whats-hot-on-your-pages.md
Comment (clmedders) [reply content/news/2014/04/2014-04-04-heatmapping-tools-show-whats-hot-on-your-pages.md:1]: updated

File: content/news/2014/04/2014-04-14-noaa-national-ocean-service-usability-case-study.md
Comment (clmedders) [reply content/news/2014/04/2014-04-14-noaa-national-ocean-service-usability-case-study.md:1]: updated

File: content/news/2014/05/2014-05-20-packaging-up-api-usability-testing-for-agency-reuse.md
Comment (clmedders) [reply content/news/2014/05/2014-05-20-packaging-up-api-usability-testing-for-agency-reuse.md:1]: updated

File: content/news/2014/05/2014-05-20-trends-on-tuesday-larger-screen-smartphones-see-accelerated-shipments.md
Comment (clmedders) [reply content/news/2014/05/2014-05-20-trends-on-tuesday-larger-screen-smartphones-see-accelerated-shipments.md:1]: updated

File: content/news/2014/05/2014-05-21-the-road-to-better-websites-gets-easier-with-usability-walkthroughs.md
Comment (clmedders) [reply content/news/2014/05/2014-05-21-the-road-to-better-websites-gets-easier-with-usability-walkthroughs.md:1]: updated

File: content/news/2014/06/2014-06-06-consumer-action-handbook-usability-case-study.md
Comment (clmedders) [reply content/news/2014/06/2014-06-06-consumer-action-handbook-usability-case-study.md:1]: updated

File: content/news/2014/10/2014-10-06-user-acceptance-testing-versus-usability-testing-whats-the-dif.md
Comment (clmedders) [reply content/news/2014/10/2014-10-06-user-acceptance-testing-versus-usability-testing-whats-the-dif.md:1]: updated

File: content/news/2014/11/2014-11-13-world-usability-day-engagement.md
Comment (clmedders) [reply content/news/2014/11/2014-11-13-world-usability-day-engagement.md:1]: updated

File: content/news/2014/11/2014-11-25-help-shape-public-participation.md
Comment (clmedders) [reply content/news/2014/11/2014-11-25-help-shape-public-participation.md:1]: updated

File: content/news/2014/12/2014-12-16-trends-on-tuesday-smartphone-users-still-use-the-phone-for-talking.md
Comment (clmedders) [reply content/news/2014/12/2014-12-16-trends-on-tuesday-smartphone-users-still-use-the-phone-for-talking.md:1]: updated

File: content/news/2015/02/2015-02-06-top-task-usability-design-for-your-users.md
Comment (clmedders) [reply content/news/2015/02/2015-02-06-top-task-usability-design-for-your-users.md:1]: updated

File: content/news/2015/02/2015-02-12-ifttt-combines-social-media-mobile-and-internet-of-things-for-government.md
Comment (clmedders) [reply content/news/2015/02/2015-02-12-ifttt-combines-social-media-mobile-and-internet-of-things-for-government.md:1]: updated

File: content/news/2015/02/2015-02-25-avoid-weak-links-in-your-digital-chain.md
Comment (clmedders) [reply content/news/2015/02/2015-02-25-avoid-weak-links-in-your-digital-chain.md:1]: updated

File: content/news/2015/04/2015-04-10-usability-testing-with-people-who-have-vision-impairment-is-difficult-reality-or-perception.md
Comment (clmedders) [reply content/news/2015/04/2015-04-10-usability-testing-with-people-who-have-vision-impairment-is-difficult-reality-or-perception.md:1]: updated

File: content/news/2015/04/2015-04-21-trends-on-tuesday-the-total-market-approach.md
Comment (clmedders) [reply content/news/2015/04/2015-04-21-trends-on-tuesday-the-total-market-approach.md:1]: updated

File: content/news/2015/06/2015-06-05-monthly-theme-building-evaluating-and-improving-government-services-through-social-media.md
Comment (clmedders) [reply content/news/2015/06/2015-06-05-monthly-theme-building-evaluating-and-improving-government-services-through-social-media.md:1]: updated

File: content/news/2015/07/2015-07-17-is-it-time-to-try-medium-new-federal-friendly-terms-of-service.md
Comment (clmedders) [reply content/news/2015/07/2015-07-17-is-it-time-to-try-medium-new-federal-friendly-terms-of-service.md:1]: updated

File: content/news/2015/08/2015-08-20-facts-on-the-federal-compatible-terms-of-service-agreement-for-yelp.md
Comment (clmedders) [reply content/news/2015/08/2015-08-20-facts-on-the-federal-compatible-terms-of-service-agreement-for-yelp.md:1]: updated

File: content/news/2015/08/2015-08-25-digitalgov-podcast-the-secrets-behind-cia-social-media.md
Comment (clmedders) [reply content/news/2015/08/2015-08-25-digitalgov-podcast-the-secrets-behind-cia-social-media.md:1]: updated

File: content/news/2015/11/2015-11-09-how-ux-effects-change-in-government-one-test-one-customer-survey-at-a-time.md
Comment (clmedders) [reply content/news/2015/11/2015-11-09-how-ux-effects-change-in-government-one-test-one-customer-survey-at-a-time.md:1]: updated

File: content/news/2015/11/2015-11-20-ux-vs-cx-whats-the-dif-part-2.md
Comment (clmedders) [reply content/news/2015/11/2015-11-20-ux-vs-cx-whats-the-dif-part-2.md:1]: updated

File: content/news/2016/02/2016-02-10-paying-incentives-for-federal-user-research.md
Comment (clmedders) [reply content/news/2016/02/2016-02-10-paying-incentives-for-federal-user-research.md:1]: updated

File: content/news/2016/04/2016-04-05-additional-data-options-with-dap.md
Comment (clmedders) [reply content/news/2016/04/2016-04-05-additional-data-options-with-dap.md:1]: updated

File: content/news/2016/05/2016-05-02-redesigning-we-the-people.md
Comment (clmedders) [reply content/news/2016/05/2016-05-02-redesigning-we-the-people.md:1]: updated

File: content/news/2016/07/2016-07-11-how-to-leverage-user-generated-content-to-resonate-with-your-audience.md
Comment (clmedders) [reply content/news/2016/07/2016-07-11-how-to-leverage-user-generated-content-to-resonate-with-your-audience.md:1]: updated

File: content/news/2016/09/2016-09-28-the-new-vote-gov-leaner-faster-and-multi-lingual.md
Comment (clmedders) [reply content/news/2016/09/2016-09-28-the-new-vote-gov-leaner-faster-and-multi-lingual.md:1]: content-strategy is present in front matter

File: content/news/2017/09/2017-09-28-finding-usability-testers-tips-from-an-army-recruiter.md
Comment (clmedders) [reply content/news/2017/09/2017-09-28-finding-usability-testers-tips-from-an-army-recruiter.md:1]: no user-testing topics present in front matter

File: content/news/2021/08/2021-08-13-a-user-centered-approach-simplicity-and-complexity-in-a-homepage-graphic.md
Comment (clmedders) [reply content/news/2021/08/2021-08-13-a-user-centered-approach-simplicity-and-complexity-in-a-homepage-graphic.md:1]: updated

File: content/news/2022/06/2022-06-13-dhs-usability-testing-kit.md
Comment (clmedders) [reply content/news/2022/06/2022-06-13-dhs-usability-testing-kit.md:1]: updated

File: content/news/2023/10/2023-10-11-5-things-we-learned-from-our-scams-wizard-usability-test.md
Comment (clmedders) [reply content/news/2023/10/2023-10-11-5-things-we-learned-from-our-scams-wizard-usability-test.md:1]: updated

File: content/resources/department-of-homeland-security-usability-testing-kit.md
Comment (clmedders) [reply content/resources/department-of-homeland-security-usability-testing-kit.md:1]: updated

File: content/resources/digitalgov-user-experience-program-results-and-case-studies.md
Comment (clmedders) [reply content/resources/digitalgov-user-experience-program-results-and-case-studies.md:1]: updated

File: content/resources/government-usability-case-studies.md
Comment (clmedders) [reply content/resources/government-usability-case-studies.md:1]: updated

File: themes/digital.gov/layouts/partials/core/get_author_image.html
Comment (clmedders) [reply themes/digital.gov/layouts/partials/core/get_author_image.html:1]: updated

**Review by clmedders — APPROVED** [review]
tested after updated and looks good

**Review by clmedders — APPROVED** [review]
tested after updated and looks good

**Review by mejiaj — APPROVED** [review]
Removed additional duplicates & cleaned up spacing in 447e8b2.

**Review by clmedders — APPROVED** [review]
looks good... retested the same steps and dont see any old topics and couldnt find any repeated topics in front matter as well

**Pull request merged**